curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646afe22e06a520335f6fffa.md

```diff
@@ -0,0 +1,106 @@
+---
+id: 646afe22e06a520335f6fffa
+title: Step 11
+challengeType: 0
+dashedName: step-11
+---
+
+# --description--
+
+Inside the CSS file, target the class selector named `title-container` and:
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed